idea/frontend/src/features/voucher/consts.ts

```diff
@@ -24,4 +24,9 @@ const DEFAULT_FILTER_VALUES = {
   status: '',
 };
 
-export { FIELD_NAME, VOUCHER_TYPE, DEFAULT_VALUES, DEFAULT_FILTER_VALUES };
+const METHOD = {
```

Comment:
Since it's a refactoring, let's move api related code to `/api` folder. You can take a look to `features/sails/api` as an example of it's implementation that recently we're trying to stick to.